### PR TITLE
[Projek] Best_Of series

### PR DESCRIPTION
**Pull request by RaikireHiuduo**
merge.

https://github.com/RaikireHiuduo/osu-wiki/tree/bo/wiki/Best_Of

- [x] 2009 (image only)
- [x] 2010 (text table)
- [x] 2011
- [x] 2012 (done before)
- [x] 2013
- [x] 2014
- [x] 2015
- [x] 2016

TODO:-
- [x] crown images.

~~TODO for after merge test:-~~
- [x] check 2015
  - ~~check for stupidity formatting (like `[ )` or `( ]` unless the player is really named so which I strongly doubt).~~

P.S. Because I am using C++, expect incomprehensible text because source copy is in ANSI encoding (Notepad default, and C++ need some external logic to work with Unicode). So, unless someone caught the encoding stupidity (like TSU?A?.BA?A?.SA?A?), I am not going to fix it.

### CODE REVIEW

File: wiki/Best_Of/2016/en.md [hunk 1]
```diff
@@ -0,0 +1,1865 @@
...
+
+## General
+
+- [News Results](https://osu.ppy.sh/news/157306073998)
+
+### Weighting formula
+
+This year’s weighted score was calculated taking into account all the scores the voter has on all the beatmaps within the given set. For each score, we use the following formula to calculate a weighting:
+
+![Formula of calculating a weighting](2016formula.png "Formula of calculating a weighting")
```
Comment (oralekin) [inline wiki/Best_Of/2016/en.md]: I don't know about the new article but I believe this should be `Formula for calculating weighting`

File: wiki/Best_Of/2016/en.md [hunk 2]
```diff
@@ -0,0 +1,1865 @@
...
+†The difficulty here takes into account the mods the player had active when achieving the score
+
+We then use the highest of those weightings (up to a maximum of 3) as the vote’s weighting. If the resultant weighting is below 0.2 (or if the player hasn’t actually passed any of the beatmaps within the set), we use 0.2 as the weighting.
+
+In short, the weighting takes into account the amount of times you’ve played a map, with it’s factor increasing on a logarithmic scale (inversely exponential). The playcount component then uses a ratio of player skill over map difficulty (taking into account the mods they had active) to determine the final weighting. This means, for the first time ever, player skill and playcount was a real factor in the scoring!
+
+## osu!standard
+
+- Full list of osu!standard 2016 beatmaps (vote screen): <https://osu.ppy.sh/community/contests/39>
+- Official Google Doc results: <https://docs.google.com/spreadsheets/d/12Xs9u7dzd5qHxkC_XE77NYjnM7xbnh9FTSMmpIJJkDE/edit#gid=0>
```
Comment (oralekin) [inline wiki/Best_Of/2016/en.md]: ¿`of the results`? 
Same thing for the other 3 modes

File: wiki/Best_Of/en.md [hunk 3]
```diff
@@ -7,277 +6,314 @@ Index page of all the "Best Of" series.
...
-          -    [Part 2](https://www.mediafire.com/?d1bi0b447zrb70r)
-     -    osu!mirror
-          -    [Part 1](https://osu.yas-online.net/tagthis.php?tag=R20)
-          -    [Part 2](https://osu.yas-online.net/tagthis.php?tag=R21)
--    [News](https://osu.ppy.sh/forum/viewtopic.php?f=9&amp;t=21059)
--    [Results](https://osu.ppy.sh/forum/t/22162)
+- [Top 30](https://osu.ppy.sh/forum/t/22162)
+- [Charts](https://osu.ppy.sh/p/chart?ch=best2009)
+- Download:
+  - Mediafire
```
Comment (oralekin) [inline wiki/Best_Of/en.md]: I believe we can remove the mediafire level from the lists as ots the only point


just remove the mediafire .-. 


same with everything else

**Comment by RaikireHiuduo** [comment]
Review request:-

1. Check link sanity (as in, are they linking or not and if it even does, is it linking to the correct place).
2. Formatting of the page.

File: wiki/Best_Of/2012/en.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 **_[Click here to return to the Best of main page](../ "Best Of main page")_**
```
Comment (MegaApplePi) [inline wiki/Best_Of/2012/en.md]: use `_Main page: [Best Of](/wiki/Best_Of "Best Of")_`

File: wiki/Best_Of/2012/en.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
...
 
 - Full list of 2012 beatmaps (vote screen): <http://osu.ppy.sh/p/bestof2012> [Legacy link. Content has been removed]
```
Comment (MegaApplePi) [inline wiki/Best_Of/2012/en.md]: 🤔 might as well get rid of it?

File: wiki/Best_Of/2013/en.md [hunk 6]
```diff
@@ -0,0 +1,1390 @@
+# 2013
+
+**_[Click here to return to the Best of main page](../ "Best Of main page")_**
```
Comment (MegaApplePi) [inline wiki/Best_Of/2013/en.md]: use `_Main page: [Best Of](/wiki/Best_Of "Best Of")_`

File: wiki/Best_Of/2014/en.md [hunk 7]
```diff
@@ -0,0 +1,1444 @@
+# 2014
+
+**_[Click here to return to the Best of main page](../ "Best Of main page")_**
```
Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: use `_Main page: [Best Of](/wiki/Best_Of "Best Of")_`

File: wiki/Best_Of/2013/en.md [hunk 8]
```diff
@@ -0,0 +1,1390 @@
...
+|498|21|[Coldplay & Rihanna - Princess of China (narakucrimson)](http://osu.ppy.sh/s/131221/ "Coldplay & Rihanna - Princess of China (narakucrimson)")|
+|499|20|[Luxion - High-Priestess (RikiH_)](http://osu.ppy.sh/s/93398/ "Luxion - High-Priestess (RikiH_)")|
+|500|20|[Tamura Yukari - W:Wonder tale (TV Size) (koreapenguin)](http://osu.ppy.sh/s/93747/ "Tamura Yukari - W:Wonder tale (TV Size) (koreapenguin)")|
+|501|20|[yanaginagi - Vidro Moyou (QL Tell Your World Mashup) (- t e n n y a -)](http://osu.ppy.sh/s/62440/ "yanaginagi - Vidro Moyou (QL Tell Your World Mashup) (- t e n n y a -)")|
+|502|20|[yanaginagi - Zoetrope (TV Size) (Ryafuka)](http://osu.ppy.sh/s/73155/ "yanaginagi - Zoetrope (TV Size) (Ryafuka)")|
+|503|20|[Spor - That's That (feat. Illy Emcee) (TicClick)](http://osu.ppy.sh/s/38929/ "Spor - That's That (feat. Illy Emcee) (TicClick)")|
+|504|20|[paraoka - petal-go-round (Irreversible)](http://osu.ppy.sh/s/110820/ "paraoka - petal-go-round (Irreversible)")|
+|505|20|[kemu - Haiboku no Shounen (Midasmens)](http://osu.ppy.sh/s/67554/ "kemu - Haiboku no Shounen (Midasmens)")|
+|506|20|[Rita - Haru Hana no Iro -Reboot Ver.- (Frostmo
```
Comment (MegaApplePi) [inline wiki/Best_Of/2013/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2013/en.md]: something broke here :/

File: wiki/Best_Of/2014/en.md [hunk 9]
```diff
@@ -0,0 +1,1444 @@
...
+|357|162|[Doxx & Feint - Mind In Motion (Secretpipe)](http://osu.ppy.sh/s/222565/ "Doxx & Feint - Mind In Motion (Secretpipe)")|
+|358|161|[Jeff Williams - I Burn (Extended Remix feat. Casey Lee Williams) (happy623)](http://osu.ppy.sh/s/137196/ "Jeff Williams - I Burn (Extended Remix feat. Casey Lee Williams) (happy623)")|
+|359|161|[P*Light - TRIGGER*HAPPY (Kyubey)](http://osu.ppy.sh/s/131361/ "P*Light - TRIGGER*HAPPY (Kyubey)")|
+|360|160|[SAKURA*TRICK - Won(*3*)Chu KissMe! (TV Size) (Andrea)](http://osu.ppy.sh/s/143362/ "SAKURA*TRICK - Won(*3*)Chu KissMe! (TV Size) (Andrea)")|
+|361|159|[REDALiCE Feat. Ayumi Nomiya - Little Star (lil_yumika_lil)](http://osu.ppy.sh/s/134998/ "REDALiCE Feat. Ayumi Nomiya - Little Star (lil_yumika_lil)")|
+|362|159|[T.M.Revolution x Mizuki Nana - Preserved Roses -Anime Version- (Flask)](http://osu.ppy.sh/s/112416/ "T.M.Revolution x Mizuki Nana - Preserved Roses -Anime Version- (Flask)")|
+|363|159|[ESTi - HELIX (xeona)](http://osu.ppy.sh/s/137037/ "ESTi - HELIX (xeona)")|
+|364|159|[akari - boys, be "stand up"!! (Hollow Wings)](http://osu.ppy.sh/s/177585/ "akari - boys, be "stand up"!! (Hollow Wings)")|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

File: wiki/Best_Of/2014/en.md [hunk 10]
```diff
@@ -0,0 +1,1444 @@
...
+|368|156|[u's - Sore wa Bokutachi no Kiseki (TV Size) (happy623)](http://osu.ppy.sh/s/163608/ "u's - Sore wa Bokutachi no Kiseki (TV Size) (happy623)")|
+|369|156|[Gavin G - Reach Out (Secretpipe)](http://osu.ppy.sh/s/199335/ "Gavin G - Reach Out (Secretpipe)")|
+|370|156|[YUZU - REASON (Trust)](http://osu.ppy.sh/s/163012/ "YUZU - REASON (Trust)")|
+|371|156|[Ryszard Rynkowski - Zycie Jest Nowela (Krfawy)](http://osu.ppy.sh/s/160008/ "Ryszard Rynkowski - Zycie Jest Nowela (Krfawy)")|
+|372|155|[yuikonnu - Natsu no Owari, Koi no Hajimari (xChippy)](http://osu.ppy.sh/s/206335/ "yuikonnu - Natsu no Owari, Koi no Hajimari (xChippy)")|
+|373|155|[Hanatan - Uta ni Katachi wa Nai keredo (Rakuen)](http://osu.ppy.sh/s/181430/ "Hanatan - Uta ni Katachi wa Nai keredo (Rakuen)")|
+|374|154|[yanaginagi - Tokohana (TV Size) (Sharlo)](http://osu.ppy.sh/s/172907/ "yanaginagi - Tokohana (TV Size) (Sharlo)")|
+|375|154|[MiddleIsland - Piano Concerto No.1 "Scream" (apaffy)](http://osu.ppy.sh/s/110126/ "MiddleIsland - Piano Concerto No.1 "Scream" (apaffy)")|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

File: wiki/Best_Of/2014/en.md [hunk 11]
```diff
@@ -0,0 +1,1444 @@
...
+|417|138|[Zhou Li Ming - Pi Li Pa La (bakabaka)](http://osu.ppy.sh/s/136065/ "Zhou Li Ming - Pi Li Pa La (bakabaka)")|
+|418|137|[Subformat - More (feat. Charli Brix) (Secretpipe)](http://osu.ppy.sh/s/184800/ "Subformat - More (feat. Charli Brix) (Secretpipe)")|
+|419|136|[IOSYS - Miracle-Hinacle (Rare)](http://osu.ppy.sh/s/158546/ "IOSYS - Miracle-Hinacle (Rare)")|
+|420|136|[Nanahira - Monosugoi Ikioide Keine ga Monosugoi Uta (Oracle)](http://osu.ppy.sh/s/140115/ "Nanahira - Monosugoi Ikioide Keine ga Monosugoi Uta (Oracle)")|
+|421|136|[Mikakuning! - Masshiro World (TV Size) (P A N)](http://osu.ppy.sh/s/144269/ "Mikakuning! - Masshiro World (TV Size) (P A N)")|
+|422|135|[Pharrell Williams - Happy (CDFA)](http://osu.ppy.sh/s/170540/ "Pharrell Williams - Happy (CDFA)")|
+|423|135|[bibuko - Sorairo Gahou (val0108)](http://osu.ppy.sh/s/106010/ "bibuko - Sorairo Gahou (val0108)")|
+|424|135|[Ray - ebb and flow (TV Size) (-Umi-)](http://osu.ppy.sh/s/141284/ "Ray - ebb and flow (TV Size) (-Umi-)")|
+|425|135|[Team "Hanayamata" - Hana wa Odore ya Irohaniho (TVsize) (Momochikun)](http://osu.ppy.sh/s/191314/ "Team "Hanayamata" - Hana wa Odore ya Irohaniho (TVsize) (Momochikun)")|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

File: wiki/Best_Of/2014/en.md [hunk 12]
```diff
@@ -0,0 +1,1444 @@
...
+|465|119|[GARNiDELiA - grilletto (TV Size Ver.) (HabiHolic)](http://osu.ppy.sh/s/197277/ "GARNiDELiA - grilletto (TV Size Ver.) (HabiHolic)")|
+|466|119|[David Guetta ft. Kid Cudi - Memories (mancuso_JM_)](http://osu.ppy.sh/s/119977/ "David Guetta ft. Kid Cudi - Memories (mancuso_JM_)")|
+|467|118|[DECO*27 feat. GUMI - Ame Moyou (sheela901)](http://osu.ppy.sh/s/189650/ "DECO*27 feat. GUMI - Ame Moyou (sheela901)")|
+|468|118|[-45 - A (Urushi38)](http://osu.ppy.sh/s/137294/ "-45 - A (Urushi38)")|
+|469|118|[Anan Ryoko - Refrain (Narcissu)](http://osu.ppy.sh/s/111386/ "Anan Ryoko - Refrain (Narcissu)")|
+|470|117|[yanaginagi - Tokohana (HabiHolic)](http://osu.ppy.sh/s/168201/ "yanaginagi - Tokohana (HabiHolic)")|
+|471|117|[SAKURA*TRICK - Won(*3*)Chu KissMe! (TV Size) (Sekai-nyan)](http://osu.ppy.sh/s/141757/ "SAKURA*TRICK - Won(*3*)Chu KissMe! (TV Size) (Sekai-nyan)")|
+|472|117|[Team "Hanayamata" - Hana wa Odore ya Irohaniho (TVsize) (Momovely)](http://osu.ppy.sh/s/191246/ "Team "Hanayamata" - Hana wa Odore ya Irohaniho (TVsize) (Momovely)")|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

File: wiki/Best_Of/2014/en.md [hunk 13]
```diff
@@ -0,0 +1,1444 @@
...
+|500|108|[SPYAIR - JUST ONE LIFE (HabiHolic)](http://osu.ppy.sh/s/151320/ "SPYAIR - JUST ONE LIFE (HabiHolic)")|
+|501|108|[Harada Hitomi - Kyoumei No True Force (NoHitter)](http://osu.ppy.sh/s/192725/ "Harada Hitomi - Kyoumei No True Force (NoHitter)")|
+|502|108|[Akiyama Uni - Chi no Iro wa Kiiro (Hollow Wings)](http://osu.ppy.sh/s/119779/ "Akiyama Uni - Chi no Iro wa Kiiro (Hollow Wings)")|
+|503|107|[coldrain - We're not alone (Lanturn)](http://osu.ppy.sh/s/134195/ "coldrain - We're not alone (Lanturn)")|
+|504|107|[Goose house - Oto no Naru Hou e (TV Size) (IamKwaN)](http://osu.ppy.sh/s/143270/ "Goose house - Oto no Naru Hou e (TV Size) (IamKwaN)")|
+|505|106|[Ray - ebb and flow (tamame's 5 years after remix) (Andrea)](http://osu.ppy.sh/s/163756/ "Ray - ebb and flow (tamame's 5 years after remix) (Andrea)")|
+|506|106|[Yooh - Dynasty (Sylith)](http://osu.ppy.sh/s/149648/ "Yooh - Dynasty (Sylith)")|
+|507|106|[Infinite Stratos Cast - BEAUTIFUL SKY (Kibbleru)](http://osu.ppy.sh/s/196241/ "Infinite Stratos Cast - BEAUTIFUL SKY (Kibbleru)")|
+|508|10
```
Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2014/en.md]: something broke here :/

File: wiki/Best_Of/2015/en.md [hunk 14]
```diff
@@ -0,0 +1,2109 @@
+# 2015
+
+**_[Click here to return to the Best of main page](../ "Best Of main page")_**
```
Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: use `_Main page: [Best Of](/wiki/Best_Of "Best Of")_`

File: wiki/Best_Of/2015/en.md [hunk 15]
```diff
@@ -0,0 +1,2109 @@
...
+- First time in the series to use a weighting system.
+- [News Results](https://osu.ppy.sh/news/138781453998)
+
+## osu!standard
+
+- Official Google Doc of the results: <https://docs.google.com/spreadsheets/d/1l_8ur1YkaboLx_7bQb70Tgkjg8c7ObyAjmJyWDBM1C0/pubhtml#>
+
+|No.|Beatmap ID|Title|Weighted Score|Raw Votes|Total Plays (Voters)|
+|:---:|:---|:---|:---:|---:|---:|
+|1|332532|[Panda Eyes & Teminite - Highscore [Fort]](https://osu.ppy.sh/s/332532/ "Panda Eyes & Teminite - Highscore [Fort]")|1.72|9817|29272|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: :-1: for using square brackets for creator
_(suppressing this for all other instances)_

File: wiki/Best_Of/2015/en.md [hunk 16]
```diff
@@ -0,0 +1,2109 @@
...
+|277|371569|[Minamotoya feat. Kuroa* - Hana Kagerou [DreaM117er]](https://osu.ppy.sh/s/371569/ "Minamotoya feat. Kuroa* - Hana Kagerou [DreaM117er]")|1.00|476|9855|
+|278|283077|[BiBi - Love Novels [Inyuschan]](https://osu.ppy.sh/s/283077/ "BiBi - Love Novels [Inyuschan]")|1.00|81|2676|
+|279|240761|[ONE OK ROCK - Liar [Snepif]](https://osu.ppy.sh/s/240761/ "ONE OK ROCK - Liar [Snepif]")|1.00|502|10945|
+|280|291517|[Demetori - Furuki Yuanxian ~ Death Echo [jonathanlfj]](https://osu.ppy.sh/s/291517/ "Demetori - Furuki Yuanxian ~ Death Echo [jonathanlfj]")|1.00|472|9698|
+|281|124857|[M2U - Masquerade [Melt]](https://osu.ppy.sh/s/124857/ "M2U - Masquerade [Melt]")|1.00|431|9736|
+|282|224335|[AKINO with bless4 - EXTRA MAGIC HOUR [koreapenguin]](https://osu.ppy.sh/s/224335/ "AKINO with bless4 - EXTRA MAGIC HOUR [koreapenguin]")|1.00|204|5927|
+|283|248979|[LiSA - Shirushi [Loneight]](https://osu.ppy.sh/s/248979/ "LiSA - Shirushi [Loneight]")|1.00|158|3195|
+|284|183623|[#ERROR!](https://osu.ppy.sh/s/183623/ "#ERROR!")|1.00|48|1199|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: ~~something broke here :/~~ the link text is wrong

File: wiki/Best_Of/2015/en.md [hunk 17]
```diff
@@ -0,0 +1,2109 @@
...
+|426|175655|[Chata - SKIP [Asagi]](https://osu.ppy.sh/s/175655/ "Chata - SKIP [Asagi]")|0.98|60|1944|
+|427|307089|[27.4beats - a zavtra ty uydyosh' [Pjecoo]](https://osu.ppy.sh/s/307089/ "27.4beats - a zavtra ty uydyosh' [Pjecoo]")|0.98|45|951|
+|428|320970|[F-777 - Wander of Thought [RikiH_]](https://osu.ppy.sh/s/320970/ "F-777 - Wander of Thought [RikiH_]")|0.98|49|1805|
+|429|276967|[Feint - Homebound [deetz]](https://osu.ppy.sh/s/276967/ "Feint - Homebound [deetz]")|0.98|347|9738|
+|430|298139|[ASSEYLUM VERS ALLUSIA starring SORA AMAMIYA - Harmonious -Short Ver.- [TT Mouse]](https://osu.ppy.sh/s/298139/ "ASSEYLUM VERS ALLUSIA starring SORA AMAMIYA - Harmonious -Short Ver.- [TT Mouse]")|0.98|52|2134|
+|431|204399|[ALiCE'S EMOTiON - Cold Rain [Yauxo]](https://osu.ppy.sh/s/204399/ "ALiCE'S EMOTiON - Cold Rain [Yauxo]")|0.98|203|7368|
+|432|239787|[Star Driver - 5 Centimeters Per Second [Come[Back]Home]](https://osu.ppy.sh/s/239787/ "Star Driver - 5 Centimeters Per Second [Come[Back]Home]")|0.98|194|5690|
+|433|268851|[AKINO from bless4 - MIIRO [xChippy]](https://osu.ppy.sh
```
Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: ~~something broke here :/~~ the grave mark broke it

Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: ~~something broke here :/~~ the pipe characters broke it

Comment (MegaApplePi) [inline wiki/Best_Of/2015/en.md]: ~~something broke here :/~~ the pipe characters broke it

File: wiki/Best_Of/2016/en.md [hunk 18]
```diff
@@ -0,0 +1,1865 @@
+# 2016
+
+**_[Click here to return to the Best of main page](../ "Best Of main page")_**
```
Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: use `_Main page: [Best Of](/wiki/Best_Of "Best Of")_`

File: wiki/Best_Of/2016/en.md [hunk 19]
```diff
@@ -0,0 +1,1865 @@
...
+|110|476697|[REOL - ChiruChiru by Pho](https://osu.ppy.sh/s/476697 "REOL - ChiruChiru by Pho")|479|326.127|
+|111|319375|[Birdy - Wings (Nu:Logic Remix) by Secretpipe](https://osu.ppy.sh/s/319375 "Birdy - Wings (Nu:Logic Remix) by Secretpipe")|337|324.992|
+|112|515069|[RADWIMPS - Yume Tourou by NoHitter](https://osu.ppy.sh/s/515069 "RADWIMPS - Yume Tourou by NoHitter")|493|318.894|
+|113|392215|[IOSYS - Cirno no Perfect Sansuu Kyoushitsu by alacat](https://osu.ppy.sh/s/392215 "IOSYS - Cirno no Perfect Sansuu Kyoushitsu by alacat")|367|317.227|
+|114|277421|[Lindsey Stirling - Senbonzakura by MrSergio](https://osu.ppy.sh/s/277421 "Lindsey Stirling - Senbonzakura by MrSergio")|452|314.292|
+|115|427508|[Wagakki Band - Senbonzakura by pkk](https://osu.ppy.sh/s/427508 "Wagakki Band - Senbonzakura by pkk")|481|313.899|
+|116|494768|[Aqours - Aozora Jumping Heart by Lami](https://osu.ppy.sh/s/494768 "Aqours - Aozora Jumping Heart by Lami")|326|312.297|
+|117|441459|[t+pazolite - Electric "Sister" Bitch by SnowNiNo_](https://osu.ppy.sh/s/441459 "t+pazolite - Electric "Sister" Bitch by SnowNiNo_")|357|311.375|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

File: wiki/Best_Of/2016/en.md [hunk 20]
```diff
@@ -0,0 +1,1865 @@
...
+|269|440897|[EGOIST - KABANERI OF THE IRON FORTRESS (TV Edit) by Meg](https://osu.ppy.sh/s/440897 "EGOIST - KABANERI OF THE IRON FORTRESS (TV Edit) by Meg")|186|139.523|
+|270|494225|[Teminite - Beastmode by Yamicchi](https://osu.ppy.sh/s/494225 "Teminite - Beastmode by Yamicchi")|215|138.437|
+|271|414348|[Petit Rabbit's - No Poi! (nenpulse bootleg remix) by Skystar](https://osu.ppy.sh/s/414348 "Petit Rabbit's - No Poi! (nenpulse bootleg remix) by Skystar")|203|138.012|
+|272|389343|[CINDERELLA PROJECT - M@GIC* by Lami](https://osu.ppy.sh/s/389343 "CINDERELLA PROJECT - M@GIC* by Lami")|136|137.853|
+|273|393343|[96Neko x KurousaP - Kagen no Tsuki by jonathanlfj](https://osu.ppy.sh/s/393343 "96Neko x KurousaP - Kagen no Tsuki by jonathanlfj")|181|137.615|
+|274|368060|[xi - Ascension to Heaven by Kroytz](https://osu.ppy.sh/s/368060 "xi - Ascension to Heaven by Kroytz")|620|137.14|
+|275|419588|[DragonForce - Above the Winter Moonlight by CircleChu](https://osu.ppy.sh/s/419588 "DragonForce - Above the Winter Moonlight by CircleChu")|228|136.8|
+|276|444288|[HERO - "Tenohira" by MugiwaraSekai](https://osu.ppy.sh/s/444288 "HERO - "Tenohira" by MugiwaraSekai")|174|136.792|
```
Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

File: wiki/Best_Of/2016/en.md [hunk 21]
```diff
@@ -0,0 +1,1865 @@
...
+|432|413909|[Chitose Sara - Arcadia (Short ver.) by Riven](https://osu.ppy.sh/s/413909 "Chitose Sara - Arcadia (Short ver.) by Riven")|96|74.52|
+|433|399126|[Joe Ford - Villain by Yoges](https://osu.ppy.sh/s/399126 "Joe Ford - Villain by Yoges")|106|74.353|
+|434|351828|[TOTTO - Wadatsumi by Desperate-kun](https://osu.ppy.sh/s/351828 "TOTTO - Wadatsumi by Desperate-kun")|76|74.109|
+|435|391671|[BoA - MASAYUME CHASING by -Nya-](https://osu.ppy.sh/s/391671 "BoA - MASAYUME CHASING by -Nya-")|113|73.899|
+|436|518311|[MONSTA X - Be Quiet by DeRandom Otaku](https://osu.ppy.sh/s/518311 "MONSTA X - Be Quiet by DeRandom Otaku")|115|73.704|
+|437|272333|[orangentle / Yu_Asahina - HAELEQUIN (Extended ver.) by RLC](https://osu.ppy.sh/s/272333 "orangentle / Yu_Asahina - HAELEQUIN (Extended ver.) by RLC")|124|73.597|
+|438|418922|[dj TAKA - True Blue by Monstrata](https://osu.ppy.sh/s/418922 "dj TAKA - True Blue by Monstrata")|106|73.227|
+|439|403065|[FELT - OUR SHIP by MokouSmoke](https://osu.ppy.sh/s/403065 "FELT - OUR SHIP by MokouSmoke")|88|73
```
Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

Comment (MegaApplePi) [inline wiki/Best_Of/2016/en.md]: something broke here :/

File: wiki/Best_Of/en.md [hunk 22]
```diff
@@ -1,283 +1,318 @@
...
 
 #### 2009 osu!standard
 
-| Placing | 2009 Results                                                            |
-|---------|-------------------------------------------------------------------------|
-| \#1     | [Queen - Don't Stop Me Now (m980)](https://osu.ppy.sh/s/5381)                 |
-| \#2     | [Yoko Hikasa - Don't Say "Lazy" (Dangaard)](https://osu.ppy.sh/s/6257)        |
+| Placing | 2009 Results |
+|---------|--------------|
+| \#1     | [Queen - Don't Stop Me Now (m980)](https://osu.ppy.sh/s/5381)  |
```
Comment (MegaApplePi) [inline wiki/Best_Of/en.md]: crown images?
_(suppressing next instances)_

File: wiki/Best_Of/en.md [hunk 23]
```diff
@@ -1,283 +1,318 @@
...
+- [2012](./2012 "Best of 2012")
+- [2013](./2013 "Best of 2013")
+- [2014](./2014 "Best of 2014")
+- [2015](./2015 "Best of 2015")
+- [2016](./2016 "Best of 2016")
+- 2017
+
 ### osu!standard
 
 #### 2009 osu!standard
```
Comment (MegaApplePi) [inline wiki/Best_Of/en.md]: remove `osu!standard` suffix?
(and other game mode suffixes?)

File: wiki/Best_Of/en.md [hunk 24]
```diff
@@ -1,283 +1,318 @@
...
+- [News #1](https://osu.ppy.sh/news/137019258158)
+- [News #2](https://news.ppy.sh/post/137019258158/best-of-2015-voting-now-open)
+- [Results #1](https://osu.ppy.sh/news/138781453998)
+- [Results #2](https://news.ppy.sh/post/138781453998/best-of-2015-results)
+
+#### 2016 osu!mania
+
+| Placing | 2016 Results |
+|---|---|
+| \#1 | [Camellia as "Bang Riot" - Blastix Riotz by Fresh Chicken](https://osu.ppy.sh/s/349208 "Camellia as "Bang Riot" - Blastix Riotz by Fresh Chicken")|
```
Comment (MegaApplePi) [inline wiki/Best_Of/en.md]: something broke here :/

File: wiki/Best_Of/en.md [hunk 25]
```diff
@@ -1,283 +1,318 @@
...
+| \#2 | [t+pazolite - Electric "Sister" Bitch by Verniy_Chan](https://osu.ppy.sh/s/310607 "t+pazolite - Electric "Sister" Bitch by Verniy_Chan")|
```
Comment (MegaApplePi) [inline wiki/Best_Of/en.md]: something broke here :/

**Comment by MegaApplePi** [comment]
- known issues during GitHub preview: some link text characters are confused as markdown markup
- links checked (as in checked for location): 2009, 2010, 2011, and 2012

File: wiki/Best_Of/en.md (hunk 23 above)
Comment (RaikireHiuduo) [inline wiki/Best_Of/en.md]: I am going to keep it.

I seriously do not want to scroll up/down just to check which gamemode is for this year in this line that I am modifying (sure as hell I will be pissed to redo if I moved standard to taiko and taiko to catch and then at mania realized I done it wrong).

Comment (MegaApplePi) [inline wiki/Best_Of/en.md]: ¯\\\_(ツ)\_/¯ it just seems redundant but fine by me

**Comment by RaikireHiuduo** [comment]
Research results (thanks @MegaApplePi ):-
- " is the only major offender.
  - Escaping the " will make the link work as intended (see 2015;entry 593, CTRL-F -> Metal Gear).
- ~ can be left alone (seems to give no problem, but I will go and escape it anyway).
- [] is iffy issue. I have no idea but there was no problem much.
- 2015 has some issue.

P.S. Because I am using C++, expect incomprehensible text because source copy is in ANSI encoding (Notepad default, and C++ need some external logic to work with Unicode). So, unless someone caught the encoding stupidity (like TSU?A?.BA?A?.SA?A? which should be TSU.BA.SA, unless someone actually use a _different dot type_), I am not going to fix it.

File: wiki/Best_Of/en.md [hunk 26]
```diff
@@ -1,283 +1,322 @@
+[#1]: /wiki/shared/GCrown.png "1st"
+[#2]: /wiki/shared/SCrown.png "2nd"
+[#3]: /wiki/shared/BCrown.png "3rd"
 
 # Best Of series
 
 Index page of all the "Best Of" series.
```
Comment (TPGPL) [inline wiki/Best_Of/en.md]: Index page for all of the "Best Of" series.?

File: wiki/Best_Of/en.md (hunk 23 above)
Comment (TPGPL) [inline wiki/Best_Of/en.md]: Delete game mode names for <year> headers?

Comment (RaikireHiuduo) [inline wiki/Best_Of/en.md]: but, I refuse.

Gimme a reason to do so.

**Pull request merged**